.idea/fileTemplates/WebDriverTestClass.java

```diff
@@ -46,7 +46,7 @@ public void preTest()
     @Test
     public void testSomething()
     {
-        assertTrue("Test something", true);
+        assertTrue("Failing stub test", false);
```

Comment:
I'm assuming the change to false/fail is because it will cover more test code?

Reply:
This is mostly to catch when folks commit an empty template test accidentally. Now that I think about it, I should probably update the template in gradlePlugins to match these changes. That's where these stub tests usually slip in from.